openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/SYS2/SYS2_ESC_calculation.yaml

```diff
@@ -1,71 +1,122 @@
+#       ]
+#   output:
+#     SYS2_savings_factor:
+#       [
+#         4.5
+#       ]
 
-- name: test SYS2 electricity savings
-  period: 2022
-  absolute_error_margin: 0.5
-  input:
-    SYS2_PDRS__postcode:
-      [
-        2043,
-        2321
-      ]
-    SYS2_star_rating:
-      [
-        seven_stars,
-        ten_stars
-      ]
-  output:
-    SYS2_electricity_savings:
-      [
-        6.1,
-        9.48
-      ]
+# - name: test SYS2 electricity savings
+#   period: 2022
+#   absolute_error_margin: 0.5
+#   input:
+#     SYS2_PDRS__postcode:
+#       [
+#         2043,
+#         2321
+#       ]
+#     SYS2_star_rating:
+#       [
+#         seven_stars,
+#         ten_stars
+#       ]
+#   output:
+#     SYS2_electricity_savings:
+#       [
+#         6.1,
+#         9.48
+#       ]
     
-- name: test SYS2 ESC calculation simple version
-  period: 2022
-  absolute_error_margin: 1
-  input: 
-    SYS2_electricity_savings:
-      [
-        6.1,
-        9.48
-      ]
-  output:
-    SYS2_ESC_calculation:
-      [
-        6.466,
-        10.0488
-      ]
+# - name: test SYS2 ESC calculation simple version
+#   period: 2022
+#   absolute_error_margin: 1
+#   input: 
+#     SYS2_electricity_savings:
+#       [
+#         6.1,
+#         9.48
+#       ]
+#   output:
+#     SYS2_ESC_calculation:
+#       [
+#         6.466,
+#         10.0488
+#       ]
+
+# - name: test SYS2 ESC not eligible on new installation
+#   period: 2022
+#   absolute_error_margin: 0.1
+#   input: 
+#     SYS2_electricity_savings:
+#       [
+#         15,
+#         15
+#       ]
+#     SYS2_replacement_activity:
+#       [
+#         False, #activity is a new installation
+#         True   #activity is a replacement
+#       ]
+#   output:
+#     SYS2_ESC_calculation:
+#       [
+#         0,
+#         15.9
+#       ]
 
-- name: test SYS2 ESC not eligible on new installation
+- name: test SYS2 ESC not eligible with input power below 100w and above 2500w
   period: 2022
   absolute_error_margin: 0.1
   input: 
+    SYS2_replacement_activity:
+      [
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
     SYS2_electricity_savings:
       [
+        15,
+        15,
+        15,
+        15,
         15,
         15
       ]
-    SYS2_replacement_activity:
+    SYS2_input_power_eligibility:
+      [
+        pump_under_100w,
+        pump_100w_to_599w,
+        pump_600w_to_1700w,
+        pump_1701w_to_2500w,
+        pump_2501w_to_3450w,
+        pump_more_than_3451w
+      ]
+    SYS2_pool_pump_type:
       [
-        False, #activity is a new installation
-        True   #activity is a replacement
+        fixed_speed_pool_pump,
+        fixed_speed_pool_pump,
+        fixed_speed_pool_pump,
+        fixed_speed_pool_pump,
+        fixed_speed_pool_pump,
+        fixed_speed_pool_pump
       ]
   output:
     SYS2_ESC_calculation:
       [
         0,
-        15.9
+        15.9,
+        15.9,
+        15.9,
```

Comment:
This is expected to be 0, since 
```
(input_power_eligibility == SYS2InputPowerEligibility.pump_1701w_to_2500w) * np.logical_not(pool_pump_type_single == SYS2PoolPumpType.single_speed_pool_pump),
```
returns false 

in this variable SYS2_input_power_ESCS_eligibility_int

Reply:
Okay - thanks, looking at these now.